jekyll/_cci2/next-gen-migration-guide.md

```diff
@@ -0,0 +1,85 @@
+short-title: "Migrating to next-gen Convenience Images"
+description: "Migrating to next-gen Convenience Images"
+order: 30
+version:
+- Cloud
+- Server v2.x
+---
+
+* TOC
+{:toc}
+
+
+## Overview
+
+Last year (2020) CircleCI introduced the next generation (next-gen) of Convenience Images. These new images are designed to replace the legacy Convenience Images that came out during the [announcement of CircleCI 2.0](https://circleci.com/blog/say-hello-to-circleci-2-0/), about 4 years ago. The next-gen CircleCI Convenience Images are designed from the ground up for a Continuous Integration environment. They’re designed to be faster, more efficient, and more importantly, more reliable. You can learn more about all of the features [on our blog post](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/). As we begin to deprecate the legacy images, here is information on the migration process.
+
+Moving from a legacy to next-gen image requires a change to the namespace. All legacy images have a Docker namespace of `circleci`, while next-gen images have a Docker namespace of `cimg`. For example, migrating from the legacy Ruby or Python image to the respective next-gen image can be done as follows:
+
+circleci/ruby:2.3.0 -> cimg/ruby:2.3.0
+circleci/python:3.8.4 -> cimg/python:3.8.4
+
+
+## Changes
+
+### Deprecated images
+
+Several changes are being made to existing images. The following images will be deprecated without next-gen equivalents:
+
+* `buildpack-deps`
+* `JRuby`
+* `DynamoDB`
+
+If you are using the `buildpack-deps` image, the suggestion is to use the new CircleCI Base image, `cimg/base`. For the other two images, you can install the software yourself in the base image or use a 3rd-party image instead.
+
+Also, the following image will be renamed:
+
+* The Go image will be changed from `golang` to `go`
+
+All Legacy to Next-Gen image changes are captured below in this table:
+
+
+
+| Legacy Image | Next-Gen Image |
+| --- | --- |
+| circleci/buildpack-deps | cimg/base |
+| circleci/jruby | No suggested path |
+| circleci/dynamodb | No suggested path |
+| circleci/golang | cimg/go |
+{: class="table table-striped"}
+
+### Browser Testing
+
+With legacy images, there were 4 different variant tags you could use to get browser testing for a particular image. For example, if you were doing browser testing with the Python v3.7.0 image, you might have used Docker image: circleci/python:3.7.0-browsers. These 4 tags have now been consolidated into a single tag designed to be used together with the [CircleCI Browser Tools orb](https://circleci.com/developer/orbs/orb/circleci/browser-tools).
+
+<table>
+<tr><th>Legacy Variant Tags</th><th>Next-gen Variant Tags</th></tr>
+<tr><td>-browsers</td><td rowspan=4>-browsers + Browser Tools Orb</td></tr>
+<tr><td>-browsers-legacy</td></tr>
+<tr><td>-node-browsers</td></tr>
+<tr><td>-node-browsers-legacy</td></tr>
+</table>
```

Comment:
This table needs to be formatted like the one in the section above

Reply:
It can't. The table above is a basic table so it can be Markdown formatted. This table has a rowspan which cannot be done in GitHub Flavored Markdown, which is what we use in Docs.

Reply:
I got two approvals so I am going to go ahead and merge. Please let me know if you'd like to discuss this further.